### ISSUE TITLE
URI malformed

### ISSUE THREAD

**Issue by antwang2** [website-scraper/node-website-scraper#251]
* **website-scraper version:**
3.3.1
**Please provide your full options object:**

<details>
<summary>Options</summary>

```js
{
    urls: ['https://www.nytimes.com/2017/10/15/world/asia/north-korea-hacking-cyber-sony.html?hp&action=click&pgtype=Homepage&clickSource=story-heading&module=first-column-region&region=top-news&WT.nav=top-news&mtrref=www.nytimes.com&gwh=2A0BDFF7694ED4DBF9B0B060BCDC3394&gwt=pay'],
    directory: `${docPath}`,
    onResourceError: (resource, err) => {
      console.log(`Resource ${resource} was not saved because of ${err}`);
    }


```

</details>

**What is expected result?**
Correctly formatted webpage.
**What is actual result?**
Ugly formatting due no css. Error: Resource { url: "https://a1.nyt.com/assets/article/20171009-082212/css/article/story/styles.css", filename: "css\styles.css", depth: 1 } was not saved because of URIError: URI malformed

Thanks in advance.

**Comment by s0ph1e** [comment]
Hi @antwang2 
Thanks for reporting, I'm working on fix for this issue

**Comment by s0ph1e** [comment]
Problem was fixed in version `3.3.2`

**Comment by antwang2** [comment]
Ok, thanks!
